brut.j.util/src/main/java/brut/util/OS.java

```diff
@@ -173,7 +173,7 @@ public void run() {
     }
 
     static class StreamCollector implements Runnable {
-        private final StringBuffer buffer = new StringBuffer();
+        private final StringBuilder buffer = new StringBuilder();
```

Comment:
I though I wanted a StringBuffer to properly block/handle multiple threads capturing output.